### PR TITLE
added file selection dialog for loading reference and current images

### PR DESCRIPTION
**Pull request by Rushi98**

### CODE REVIEW

File: src/sources/mainwindow.cpp [hunk 1]
```diff
@@ -13,3 +16,46 @@ MainWindow::~MainWindow()
 {
     delete ui;
 }
+
+void MainWindow::on_actionLoad_Reference_Image_triggered()
+{
+    QString fileName = QFileDialog::getOpenFileName(this,
+                                                     tr("Select Reference Image File"),
+                                                     "",
+                                                     "Images (*.png *.jpg *.tiff);;All Files (*)"
```
Comment (kukreja-vikramaditya) [inline src/sources/mainwindow.cpp]: Add default support for *.tif and *.jpeg also

File: src/sources/mainwindow.cpp [hunk 2]
```diff
@@ -13,3 +16,46 @@ MainWindow::~MainWindow()
...
+    else status = tr("Selected ") + fileName;
+    statusBar()->showMessage(status);
+}
+
+void MainWindow::on_actionLoad_Current_Image_s_triggered()
+{
+    QStringList files = QFileDialog::getOpenFileNames(this,
+                                                      tr("Select Current Image(s)"),
+                                                      "",
+                                                      "Images (*.png *.jpg *.tiff);;All Files (*)"
```
Comment (kukreja-vikramaditya) [inline src/sources/mainwindow.cpp]: same here *.jpeg and *.tif

File: src/sources/mainwindow.cpp [hunk 3]
```diff
@@ -13,3 +16,46 @@ MainWindow::~MainWindow()
...
+                                                      );
+    QString status = "";
+    switch (files.length()) {
```
Comment (kukreja-vikramaditya) [inline src/sources/mainwindow.cpp]: for multiple images just mention the number of files? Or at the most the first and the last file names.

File: src/sources/sources.pri [hunk 4]
```diff
@@ -1,3 +1,5 @@
 SOURCES += \
         src/sources/main.cpp \
         src/sources/mainwindow.cpp
+
+HEADERS +=
```
Comment (kukreja-vikramaditya) [inline src/sources/sources.pri]: Is this required?

**Review by kukreja-vikramaditya — CHANGES_REQUESTED** [review]
Minor changes to be made

File: src/sources/mainwindow.cpp [hunk 5]
```diff
@@ -13,3 +16,46 @@ MainWindow::~MainWindow()
...
+            status = status + tr("Selected ") + tr(std::to_string(files.length()).c_str()) + tr(" Files : ");
+            QList<QString>::iterator it = files.begin();
+            status += *it;
+            it++;
+            for (; it != files.end(); it++) {
+                status += ", " + *it;
+            }
+        } break;
+    }
+    statusBar()->showMessage(status);
```
Comment (kukreja-vikramaditya) [inline src/sources/mainwindow.cpp]: Also add timeout for the `showMessage` function
http://doc.qt.io/qt-5/qstatusbar.html#showMessage

**Comment by Rushi98** [comment]
Implemented review

**Review by kukreja-vikramaditya — APPROVED** [review]

**Pull request merged**